package.json

```diff
@@ -25,8 +25,7 @@
     "README.md",
     "LICENSE.md"
   ],
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "lib/index.ts",
```

Comment:
@hannojg This change is breaking the process of linking the Onyx repo to E/App one, described [here](https://github.com/Expensify/react-native-onyx?tab=readme-ov-file#development). Why do we need this change?

Reply:
Oh excuse, this was meant to be handled in a separate PR.
To be honest, i have never seen the development section in the readme yet. I think these changes are unnecessary if the DX is good with what's described there.

For react native we could however consume the typescript code directly. I think this is what i did here:

- https://github.com/Expensify/react-native-onyx/pull/603/files

in the light of the better debugging tools i think for RN it would be nicer to be able to debug the TS code in NewDot than having to debug the transpiled JS code.

Reply:
Actually these changes are also breaking the code on E/App, not just the development process described there, so I created a PR to revert it [here](https://github.com/Expensify/react-native-onyx/pull/606).

I will take a look at your PR and continue review there, thanks!